python/cudf_polars/cudf_polars/utils/config.py

```diff
@@ -99,6 +100,38 @@ def __post_init__(self) -> None:
             raise TypeError("pass_read_limit must be an int")
 
 
+def default_blocksize(scheduler: str) -> int:
```

Comment:
@pentschev - This is what we do for dask-cudf to get a reasonable default. Let me know if you think this is still a reliable approach.